### ISSUE TITLE
href-no-hash missing from rules in 3.4.0

### ISSUE THREAD

**Issue by acourdavault** [dequelabs/axe-core#1872]
Hello,

I am using Jest-axe, updated to use axe-core 3.4.0
I check in the axe.js file, and i do not find this rule, although it is reported.

I have noticed it is in the docs:
https://github.com/dequelabs/axe-core/blob/develop-2x/doc/API.md

Not in develop branch
https://github.com/dequelabs/axe-core/blob/develop/doc/API.md

Could you tell me if this rule will be added? or if it was removed

Here is the paragraph in the rules.
```javascript
        'frame-title-unique': {
          description: 'Ensures <iframe> and <frame> elements contain a unique title attribute',
          help: 'Frames must have a unique title attribute'
        },
        'frame-title': {
          description: 'Ensures <iframe> and <frame> elements contain a non-empty title attribute',
          help: 'Frames must have title attribute'
        },
        'heading-order': {
          description: 'Ensures the order of headings is semantically correct',
          help: 'Heading levels should only increase by one'
        },
        'hidden-content': {
          description: 'Informs users about hidden content.',
          help: 'Hidden content on the page cannot be analyzed'
        },
        'html-has-lang': {
          description: 'Ensures every HTML document has a lang attribute',
          help: '<html> element must have a lang attribute'
        },
```

**Comment by straker** [comment]
The `href-no-hash` was removed in 3.0.0 https://github.com/dequelabs/axe-core/blob/develop/CHANGELOG.md#300-alpha9-2018-01-18

**Comment by straker** [comment]
Closing. Please reopen if you find axe is producing an error for this rule.
